### ISSUE TITLE
Files aren't decoded correctly by the Spreadsheets Node

### ISSUE THREAD

**Issue by iosifnicolae2** [n8n-io/n8n#388]
### How to reproduce the problem?
1. Create a `.csv` file using the below content:
```
Idß,Projektß
1,test
2,test
3,test
```
2. Load the file using the `Read Binary file` Node and then connect a `Spreadsheets` Node
3. The output table will look like:
![image](https://user-images.githubusercontent.com/7108834/77094621-69cb2580-6a15-11ea-81c7-27ee46ebf99f.png)

### Solution
- change this line: https://github.com/n8n-io/n8n/blob/bcaddecdd65a78a6c1863ee1e8d574373e33a37e/packages/nodes-base/nodes/SpreadsheetFile.node.ts#L192 
to 
```javascript
const workbook = xlsxRead(binaryData.toString(), {type: "string"});
```

**Comment by janober** [comment]
Sorry, I had to do first some testing as it looked like it could cause problems.

After I did now test it turned out to really be the case as it breaks for example ".ods" files. So added it now as option "Read as String" instead. If it gets set, it does read the file as you described above. 

That will be released with the new version tomorrow.

**Comment by janober** [comment]
Fix got released with n8n@0.59.0
